### PR TITLE
Benchmark categories renaming

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Rename categories
- Fix https://github.com/Samsung/CredData/issues/71
- round benchmark scores to 10^-6
- ignore whitespaces in benchmark workflow

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -163,7 +165,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v4
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: auxiliary
```
Comment (babenek) [inline .github/workflows/benchmark.yml:169]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +63,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:67]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -22,7 +22,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/72

**Review by csh519 — APPROVED** [review]
2 Benchmark categories are renamed.

LGTM :+1:

**Review by xDizzix — APPROVED** [review]

**Pull request merged**